compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -1288,48 +1390,58 @@
       (+ m div))))
 
 ;; Numbers.
-(def integer?
+(defn integer?
   "Returns true if n is an integer"
-  clojure.core-native/integer?)
-(def float?
+  [n]
+  (cpp/jank.runtime.is_integer n))
+(defn float?
   "Returns true if n is a floating point number"
-  clojure.core-native/real?)
-(def boolean?
+  [n]
+  (cpp/jank.runtime.is_real n))
+(defn boolean?
   "Returns true if x is a boolean"
-  clojure.core-native/boolean?)
-(def number?
+  [x]
+  (cpp/jank.runtime.is_boolean x))
+(defn number?
   "Returns true if x is a number"
-  clojure.core-native/number?)
+  [x]
+  (cpp/jank.runtime.is_number x))
 
-(def even?
+(defn even?
   "Returns true if n is even, throws an exception if n is not an integer"
-  clojure.core-native/even?)
-(def odd?
+  [n]
+  (cpp/jank.runtime.is_even n))
+(defn odd?
   "Returns true if n is odd, throws an exception if n is not an integer"
-  clojure.core-native/odd?)
+  [n]
+  (cpp/jank.runtime.is_odd n))
 
 ;; Random.
 (defn rand
   "Returns a random floating point number between 0 (inclusive) and
    n (default 1) (exclusive)."
   ([]
-   (clojure.core-native/rand))
+   (cpp/jank.runtime.rand))
   ([n]
    (* (rand) n)))
 
 ;; Strings.
-(def subs
+(defn subs
   "Returns the substring of s beginning at start inclusive, and ending
    at end (defaults to length of string), exclusive."
-  clojure.core-native/subs)
+  ([s start]
+   (cpp/jank.runtime.subs s start))
+  ([s start end]
+   (cpp/jank.runtime.subs s start end)))
 
 (def ^:private first-index-of clojure.core-native/first-index-of)
```

Comment:
I think these are unused and private, should I remove them?

Reply:
Go for it!